reviews/layout/helpers.py

```diff
@@ -19,15 +19,24 @@ def render_pull_request_table(
     """Renders a list of pull requests as a table"""
     table = Table(show_header=True, header_style="bold white")
     table.add_column("#", style="dim", width=5)
-    table.add_column(title, width=60)
```

Comment:
had to change widths here so that the RTR column gets rendered properly without the cutoff with ```...```

Reply:
Could you include a screenshot in the pull request description, please?

Reply:
If you adjust this to the following: it removes some of the unused space on the left layout to free it up for usage on the right

```python
    layout["main"].split_row(
        Layout(name="left_side", size=50),
        Layout(name="body", minimum_size=80),
    )
 ```
 
 Rich documented on fixed size: https://rich.readthedocs.io/en/stable/layout.html#fixed-size
 
![image](https://user-images.githubusercontent.com/1443700/116824400-db1a5900-ab81-11eb-9870-a7d46f0eab33.png)

Reply:
I'll take a look at this, thank you very much!